internal/kafka/internal/services/kafka.go

```diff
@@ -498,6 +499,28 @@ func (k *kafkaService) ListByStatus(status ...constants.KafkaStatus) ([]*dbapi.K
 	return kafkas, nil
 }
 
+func (k *kafkaService) ListKafkasToBePromoted() ([]*dbapi.KafkaRequest, *errors.ServiceError) {
+	dbConn := k.connectionFactory.New()
+
+	validPromotionStatuses := []string{
```

Comment:
In `internal/kafka/internal/handlers/validation.go` the `validateKafkaRequestToPromoteHasAPromotableStatus` also has an instantiation of a variable called `acceptedKafkaStatusesForPromotion`.

Maybe it would be a good idea to somehow reuse the list of states for that between them